COMPILING.md

```diff
@@ -567,7 +567,7 @@ Extract the 'WinDepend' folder and put it in the root folder of Cataclysm projec
 
 ### Building
 
-Building Cataclysm with Visual Studio is very simple. Just build it like a normal Visual C++ project. The process may takes a long period of time, so you'd better prepare a cup of coffee and some books in front of your computer :)
+Building Cataclysm with Visual Studio is very simple. First create the version.h file by running the batch script `prebuild.cmd` in the folder 'msvc-full-features'.  Then just build it like a normal Visual C++ project. The process may takes a long period of time, so you'd better prepare a cup of coffee and some books in front of your computer :)
```

Comment:
I am pretty sure that `prebuild.cmd` is run automatically as prebuild event:

https://github.com/CleverRaven/Cataclysm-DDA/blob/95bfcf2380a8a1e1a2865db17bf3b397bd07c6d9/msvc-full-features/Cataclysm-vcpkg-static.vcxproj#L155-L157

Reply:
Hmm, I got a compile error and was missing version.h when I built it in VS2017

Reply:
Could be some user side settings. I will need to check that.

Reply:
It seems `prebuild.cmd` wasn't set in prebuild event only for Win32, but not for x64:

![image](https://user-images.githubusercontent.com/16213433/55140944-4d6e1300-514a-11e9-9ae8-f2952c170814.png)

![image](https://user-images.githubusercontent.com/16213433/55140907-38917f80-514a-11e9-85c2-4762df652b2f.png)

Reply:
Also you need to have `git` available on some `PATH` as version string is generated by following command:

https://github.com/CleverRaven/Cataclysm-DDA/blob/95bfcf2380a8a1e1a2865db17bf3b397bd07c6d9/msvc-full-features/prebuild.cmd#L8

I will update the script to generate some version number without git.

Reply:
@ZhilkinSerg Thanks.  I'll get rid of this comment then

Reply:
Try the files from #29070 and see if it helps.

Reply:
Will do.  I updated this PR with just a link to compiling.md